### ISSUE TITLE
Unschedulable job not correctly reconciled

### ISSUE THREAD

**Issue by mouradmourafiq** [polyaxon/polyaxon#1208]
### Describe the bug

```
Hi all. If a job is not schedulable I can't find anything pointing to that in the GUI in v1 CE (just shows me running but nothing is happening). Is that information available somewhere, or not implemented yet? (In 0.6 there was a status for that)


Hi, it should show a warning status


is the issue related to the init container?


This is how I produced this:
    environment:
      nodeSelector:
        kubernetes.io/role: blabla
If I change it to an actually existing label, it works

ok it might be a bug or  a regression

ok. thanks for the answer :slightly_smiling_face:
```

**Comment by tobigue** [comment]
Additionally, I had the case that after stopping the unschedulable runs, the "running" step does not finish and the pods are still "pending" in k8s.

![image](https://user-images.githubusercontent.com/1560152/104617861-43553580-568c-11eb-9be9-5e63cad86393.png)

**Comment by mouradmourafiq** [comment]
Indeed, thanks for the additional info. I think this is also an issue when the operation is having the warning status:

<img width="372" alt="Screenshot 2021-01-14 at 17 27 43" src="https://user-images.githubusercontent.com/1261626/104619225-e9556f80-568d-11eb-91bd-fe6b3b3e451f.png">

The underlying service in my case was not deleted.

**Comment by mouradmourafiq** [comment]
I will think we will fix this issue before 1.6 with other similar issues #1200 and #1129
